entity/DecisionRuleEntities.xml

```diff
@@ -0,0 +1,144 @@
+        </relationship>
+        <relationship type="one" fk-name="RULE_GROUP" related="co.hotwax.common.RuleGroup" short-alias="ruleGroup">
+            <key-map field-name="ruleGroupId"/>
+        </relationship>
+        <relationship type="many" related="co.hotwax.common.RuleCondition" short-alias="ruleConditions">
+            <key-map field-name="ruleId"/>
+        </relationship>
+        <relationship type="many" related="co.hotwax.common.RuleAction" short-alias="ruleActions">
+            <key-map field-name="ruleId"/>
+        </relationship>
+        <master>
+            <detail relationship="ruleConditions"><detail relationship="ruleConditionType"/></detail>
+            <detail relationship="ruleActions"><detail relationship="ruleActionType"/></detail>
+        </master>
+    </entity>
+    <entity entity-name="RuleCondition" package="co.hotwax.common" group="ofbiz_transactional" short-alias="ruleCondition">
+        <description>Defines conditions used in rules.</description>
+        <field name="ruleId" type="id" is-pk="true"><description>Unique identifier for the rule.</description></field>
+        <field name="conditionSeqId" type="id" is-pk="true"><description>Unique identifier for the rule inventory condition.</description></field>
+        <field name="conditionTypeEnumId" type="id"><description>Type of condition (Filter/ Order by)</description></field>
+        <field name="fieldName" type="text-medium" enable-audit-log="true"><description>Input parameter for the condition.</description></field>
+        <field name="operator" type="text-medium" enable-audit-log="true"><description>Operator used in the condition.</description></field>
+        <field name="fieldValue" type="text-long" enable-audit-log="true"><description>Value of the condition.</description></field>
+        <field name="multiValued" type="text-medium" enable-audit-log="true"><description>Indicator for multi value field mainly for Solr/ES schema.</description></field>
+        <field name="sequenceNum" type="number-integer" enable-audit-log="true"><description>Sequence number for ordering.</description></field>
+        <field name="createdDate" type="date-time"><description>Date and time when the condition was created.</description></field>
+        <relationship type="one" fk-name="RULE_COND_TYPEENUM" title="RuleConditionType" related="org.apache.ofbiz.common.enum.Enumeration" short-alias="ruleConditionType">
+            <key-map field-name="conditionTypeEnumId" related="enumId"/>
+        </relationship>
+        <relationship type="one" fk-name="RULE_COND_RULE" related="co.hotwax.common.DecisionRule" short-alias="decisionRule">
+            <key-map field-name="ruleId"/>
+        </relationship>
+        <master>
+            <detail relationship="ruleConditionType"/>
+        </master>
+    </entity>
+    <entity entity-name="RuleAction" package="co.hotwax.common" group="ofbiz_transactional" short-alias="ruleAction">
```

Comment:
we can use co.hotwax.rule as package name

Reply:
Done. Thanks!